UnityMcpBridge/UnityMcpServer~/src/tools/manage_script_edits.py

```
@@ -80,6 +81,37 @@ def index_of(line: int, col: int) -> int:
+                import json, glob, os
+                # Small delay so write flushes; prefer early flip to avoid editor-focus second reload
+                time.sleep(0.1)
+                try:
+                    files = sorted(glob.glob(os.path.expanduser("~/.unity-mcp/unity-mcp-status-*.json")), key=os.path.getmtime, reverse=True)
+                    if files:
+                        with open(files[0], "r") as f:
+                            st = json.loads(f.read())
+                            if st.get("reloading"):
+                                return
+                except Exception:
+                    pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)